authui/.stylelintrc.json

```diff
@@ -20,6 +16,7 @@
       {
         "ignoreAtRules": ["mixin", "include", "tailwind", "layer"]
       }
-    ]
+    ],
+    "length-zero-no-unit": null
```

Comment:
Opened this issue to follow up https://github.com/oursky/stylelint-oursky/issues/23